src/identity.cc

```diff
@@ -105,7 +105,9 @@ TRITONSERVER_Error*
 ModelState::Create(TRITONBACKEND_Model* triton_model, ModelState** state)
 {
   try {
-    *state = new ModelState(triton_model);
+    // Identity backend supports optional inputs intrinsically as it executes
+    // requests at per input basis
```

Comment:
per request basis?

Reply:
input. It only process inputs as provided by the request, not necessary requires all inputs specified in model config.